src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationTryBlock.java

```diff
@@ -0,0 +1,24 @@
+/* Config:                                                                  //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration: //indent:1 exp:1
+ *                                                                          //indent:1 exp:1
+ * basicOffset = 4                                                          //indent:1 exp:1
+ * braceAdjustment = 0                                                      //indent:1 exp:1
+ * caseIndent = 4                                                           //indent:1 exp:1
+ * forceStrictCondition = true                                              //indent:1 exp:1
+ * lineWrappingIndentation = 8                                              //indent:1 exp:1
+ * tabWidth = 4                                                             //indent:1 exp:1
+ * throwsIndent = 4                                                         //indent:1 exp:1
+ */                                                                         //indent:1 exp:1
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+import java.io.InputStreamReader; //indent:0 exp:0
+
+public class InputIndentationTryBlock { //indent:0 exp:0
+    public void delete(final int assetId) { //indent:4 exp:4
+        try { //indent:8 exp:8
+            new Object().equals(Integer.valueOf(assetId)); //indent:12 exp:12
```

Comment:
please add one more sentence from 
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/1526546_2021085309/reports/diff/Orekit/index.html#A6

to make sure it is covered.
We try to copy from regression reports all cases that are become different and it is not covered by our Inputs.

Reply:
Done